tests/test_parse/test_parse_bel_relations.py

```diff
@@ -910,7 +912,7 @@ def test_has_reaction_component(self):
         statement = 'rxn(reactants(a(CHEBI:"(S)-3-hydroxy-3-methylglutaryl-CoA"),a(CHEBI:NADPH), \
                     a(CHEBI:hydron)),products(a(CHEBI:mevalonate), a(CHEBI:"CoA-SH"), a(CHEBI:"NADP(+)"))) \
                     hasReactant a(CHEBI:"(S)-3-hydroxy-3-methylglutaryl-CoA")'
-        result = self.parser.relation.parseString(statement)
```

Comment:
I think this test is just checking that there's no exception thrown. The assignment can be removed, but the expression should stay